www/js/Widget.js

```diff
@@ -368,6 +371,15 @@ class Widget {
     }
   }
 
+  _marquee () {
+    let titleWidth = this.ele.querySelector('.w-top-bar__title').clientWidth
+    let titleSpanWidth = this.ele.querySelector('.w-top-bar__title > span').clientWidth
+    if (titleSpanWidth >= titleWidth) {
+      this.ele.querySelector('.w-top-bar__title').classList.add('marquee')
+      this.ele.querySelector('.w-top-bar__title > span').style.animationDelay = `${Math.random() * 3}s`
+    }
+  }
+
```

Comment:
hey! just tried this out + this looks great :D 
one tiny thing, could we change those `let` to `const` 
also, question, why the random animation time instead of a consistent value?

Reply:
yep yep yep, i'll change those variables during lunch or after work today!

i honestly just didnt like how it looked when multiple widget headers on a page would start scrolling at the same exact time, so this was my really quick solution for that. its purely aesthetic and i'd love ur 2 cents if u have any. not sure if its necessary to begin with, and i could imagine something relating to the widget width being better then a randomized number

Reply:
ooooooh, i see. ok, yea totally agree + think Math.random() is a great solution in this case